api/rest/v1/bulk_info_variable_group.md

```diff
@@ -0,0 +1,198 @@
+display name, a list of child variables with their information, a list of child variable groups
+with their information and the number of descendent variables. If varible groups DCIDs are not provide, then
+all the variable group information will be retrieved.
+
+<div markdown="span" class="alert alert-info" role="alert" style="color:black; font-size: 0.8em">
+   <span class="material-icons md-16">info </span><b>Tip:</b><br />
+   Variable group is used in the statistical variable hierarch UI widget as shown in [Statistical Variable Explorer](https://datacommons.org/tools/statvar).
+</div>
+
+<div markdown="span" class="alert alert-warning" role="alert" style="color:black; font-size: 0.8em">
+    <span class="material-icons md-16">info </span><b>See Also:</b><br />
+    For querying a single variable group and a simpler output, see the [simple version](/api/rest/v1/info/variable-group) of this endpoint.
+</div>
+
+## Request
+
+<div class="api-tab">
+  <button id="get-button" class="api-tablink" onclick="openTab(event, 'GET-request')">
+    GET Request
+  </button>
+  <button id="post-button" class="api-tablink" onclick="openTab(event, 'POST-request')">
+    POST Request
+  </button>
+</div>
+
+<div id="GET-request" class="api-tabcontent api-signature"><div class="scroll">
+https://api.datacommons.org/v1/bulk/info/variable-group?nodes={variable_group_dcid_1}&nodes={variable_group_dcid_2}&key={your_api_key}
+</div></div>
+
+<div id="POST-request" class="api-tabcontent api-signature"><div class="scroll">
+URL:
+https://api.datacommons.org/v1/bulk/info/variable-group
+
+Header:
+X-API-Key: {your_api_key}
+
+JSON Data:
+{
+"nodes":
+[
+"{variable_group_dcid_1}",
+"{variable_group_dcid_2}",
+...
+]
+}
+
+</div></div>
+
+<script src="/assets/js/syntax_highlighting.js"></script>
+<script src="/assets/js/api-doc-tabs.js"></script>
+
+### Path Parameters
+
+This endpoint has no path parameters.
+
+### Query Parameters
+
+| Name                                                              | Type            | Description                                                                                                                                                     |
+| ----------------------------------------------------------------- | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| key <br /> <required-tag>Required</required-tag>                  | string          | Your API key. See the [page on authentication](/api/rest/v1/getting_started#authentication) for a demo key, as well as instructions on how to get your own key. |
+| nodes <br /> <optional-tag>Optional</optional-tag>                | string          | [DCIDs](/api/rest/v1/getting_started#dcid) of the variable groups to query information for.                                                                     |
```

Comment:
Do we have a list somewhere on the website where users can find valid variable group DCIDs? Otherwise, maybe we should point them to the statVar explorer?

Reply:
Users can not really find variable group DCIDs directly from the explorer. There is no direct use case for this yet. Let me think about it and add the use scenario later.